external/jacoco/test.properties

```diff
@@ -0,0 +1,14 @@
+github_url="https://github.com/jacoco/jacoco.git"
+        script="jacoco-test.sh"
```

Comment:
Please fix the indentation so that all lines are flush with the left margin (no spaces or tabs at the front of the lines 2-13)